.github/workflows/build-ci.yml

```
@@ -0,0 +1,80 @@
+name: build-ci
+
+on:
+  pull_request:
+    paths:
+      - "**/Dockerfile"
+      - "**/docker-entrypoint.py"
+      - "**/PLATFORMS"
+      - "tests/"
+      - "tools/genmatrix.js"
+      - ".github/workflows/build-ci.yml"
+
```

For master only ? While we don't have a branch strategy
[warning indentation, to copy manually if used]
```suggestion
      - ".github/workflows/build-ci.yml"
    branches:
      - master         
```

---

This would limit the CI trigger to exclusively apply for pull requests that are raised with `main` as the base. Even if we would have more branches than `main` in this repository - I'd rather not - would we not want this to trigger when a PR is filed against, say, a split-off development branch? If not, why not?

---

Yes, not necessary a good idea